### PR TITLE
Jump to charging plan From transaction in progress 

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts [hunk 1]
```diff
@@ -210,6 +213,9 @@ export class TransactionsInProgressTableDataSource extends TableDataSource<Trans
           (actionDef as TableViewTransactionActionDef).action(transaction, this.dialog, this.refreshData.bind(this));
         }
         break;
+      case ChargingStationButtonAction.CHECK_CHARGING_PLANS:
+        this.checkChargingPlansAction.action('charging-stations#chargingplans?ChargingStationID=' + transaction.chargeBoxID + '&TransactionID=' + transaction.id);
```
Comment (LucasBrazi06) [inline src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts:217]: Why is there no downcast here?

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts (hunk 1 above)
Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts:217]: Downcast as to pass the object transaction to the action and handle it in button file ?

Comment (LucasBrazi06) [reply src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts:217]: I would expect that kind of call:

`(actionDef as TableViewTransactionActionDef).action(transaction, this.dialog, this.refreshData.bind(this));`

Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts:217]: Ok

Comment (mahdibenromdhane) [reply src/app/pages/transactions/in-progress/transactions-in-progress-table-data-source.ts]: Done.

**Pull request merged**